WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -538,20 +549,30 @@ public static int removeTagsFromPost(long blogId, long postId, final ReaderTagTy
         }
 
         String[] args = {Integer.toString(tagType.toInt()), Long.toString(blogId), Long.toString(postId)};
-        return ReaderDatabase.getWritableDb().delete(
+        int rowsDeleted = ReaderDatabase.getWritableDb().delete(
```

Comment:
Another instance of the method directly returning the int value. I changed to keep the value local and will only post the new event is rows have actually been deleted.